scripts/kconfig/kconfiglib.py

```diff
@@ -556,6 +556,16 @@ def my_other_fn(kconf, name, arg_1, arg_2, ...):
 
 VERSION = (14, 1, 0)
 
+# Bump the recursion limit from 1k to 5k. The limit was observed to be
+# broken in NCS when gen_kconfig_rest.py was executed. Specifically,
```

Comment:
Why not do this in gen_kconfig_rest.py instead, then?

Reply:
Because that would also be a noup commit. I'm trying to figure out a way to make this upstreamable.

But I had to task-switch to crypto.

Perhaps someone else is able to find a way to make this fromlist instead of noup?

Perhaps just a PR will do, maybe upstream will accept this as-is. After moving the patch to gen_kconfig_rest maybe.